README.md

```diff
@@ -112,16 +112,19 @@ skyflowClient := Skyflow.Init(configuration)
 
 All Vault APIs must be invoked using a skyflowClient instance.
 
-#### Insert
+### Insert data into the vault
+
+To insert data into your vault, use the **Insert(records map[string]interface{}, options common.InsertOptions)** method of the Skyflow client. The **insertInput** parameter requires a `records` key and takes an array of records to be inserted into the vault as a value. The `options` parameter is a InsertOptions object that provides further options, including Upsert operations, for your insert call, as shown below.
```

Comment:
Change 

> The **insertInput** parameter requires a `records` key and takes an array of records to be inserted into the vault as a value.

to active voice:

The **insertInput** parameter requires a `records` key and takes an array of records to insert as a value into the vault.

Reply:
Done